### PR TITLE
Expand/tidy comments

### PR DESCRIPTION
**Pull request by paulc4**

### CODE REVIEW

File: nuodb/env-default [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #
-# default ENV VAR values
+# Default ENV VAR values. You must copy this to .env before running docker-compose.
+# DO NOT modify this file.
```
Comment (NikTJ777) [inline nuodb/env-default:3]: We should clarify this to be: "Do not modify the 'env-default' file.

**Review by NikTJ777 — COMMENTED** [review]
Let me know if you are happy with my suggestion.

File: nuodb/env-default (hunk 1 above)
Comment (paulc4) [reply nuodb/env-default:3]: Fair point.  I'll leave it with  you to merge this request and then make that change.

**Review by NikTJ777 — APPROVED** [review]

**Comment by NikTJ777** [comment]
Nice improvements - thank you!

**Pull request merged**